runtests.py

```diff
@@ -1743,11 +1746,15 @@ def runTest(self):
                                      stdout=subprocess.PIPE,
                                      shell=True,
                                      env=env)
-                out, err = p.communicate()
+                _out, _err = p.communicate()
+                cmd.append(command)
+                out.append(_out)
+                err.append(_err)
             res = p.returncode
             if res != 0:
-                sys.stderr.write("%s\n%s\n%s\n" % (
-                    command, self._try_decode(out), self._try_decode(err)))
+                for c, o, e in zip(cmd, out, err):
+                    sys.stderr.write("%s\n%s\n%s\n\n" % (
+                        c, self._try_decode(o), self._try_decode(e)))
```

Comment:
This was already part of the other PR, right?

Reply:
yes, sorry. Removing.